Resources/Prototypes/Reagents/medicine.yml

```
@@ -794,19 +810,21 @@
   color: "#00e5ff"
   metabolisms:
     Medicine:
+      metabolismRate: 0.2 # Funkychem - rebalance existing chems
       effects:
       - !type:HealthChange
         conditions:
         - !type:TotalDamage
-          max: 50
+          max: 40 # Funkychem - rebalance existing chems
         damage:
           groups:
-            Brute: -1
+            Brute: -0.9 # Funkychem - rebalance existing chems
           types:
-            Poison: -0.5 ##Should be about what it was when it healed the toxin group
-            Heat: -0.33
-            Shock: -0.33
-            Cold: -0.33
+            # Funkychem - rebalance existing chems
+            Poison: -0.3 ##Should be about what it was when it healed the toxin group
```

Comment outdated and inaccurate
```suggestion
            Poison: -0.3
```